styles/state.css

```diff
@@ -1,13 +1,16 @@
 
```

Comment:
I would try and incorporate this into either module, layout or base. In this class we do not use state and theme! :)